tests/framework/utils.py

```diff
@@ -432,7 +432,7 @@ def run_cmd_sync(cmd, ignore_return_code=False, no_shell=False, cwd=None):
 
     # Log the message with one call so that multiple statuses
     # don't get mixed up
-    CMDLOG.debug(output_message)
+    CMDLOG.info(output_message)
```

Comment:
not sure I understand what u mean with increasing verbosity since debug is more verbose than info, right?

Reply:
Well I was thinking it as we are making the logs more verbose, by enabling this by default. I think I will have to limit the output as this can output megabytes apparently.